src/Essentials/src/Permissions/Permissions.android.cs

```diff
@@ -441,8 +441,28 @@ public partial class Speech : BasePlatformPermission
 
 		public partial class StorageRead : BasePlatformPermission
 		{
-			public override (string androidPermission, bool isRuntime)[] RequiredPermissions =>
-				new (string, bool)[] { (Manifest.Permission.ReadExternalStorage, true) };
+			public override (string androidPermission, bool isRuntime)[] RequiredPermissions
+			{
+				get
+				{
+					var permissions = new List<(string, bool)>();
+
+#if __ANDROID_33__
```

Comment:
Can we stop using the old Xamarin.Android defines and use the new .NET ones?
```suggestion
#if ANDROID33
```